README.md

```diff
@@ -1,68 +1,219 @@
+## Requirements
 
-To connect to relay just send your event to localhost at default ports 12999/13002
+To use this docker-compose file the following requirements must be met:
 
-The compose is ready to share ports from 12999 to 13030 to cover more needs than default ports
+* docker-compose version: [v1.27+](https://github.com/docker/compose/releases)
+* the following folders **must exist at the same level as *docker-compose.yaml*
+  file**:
+    * *conf*: Stores the NG-Relay configuration. The first time the Devo NG-Relay
+      is run, a default configuration is populated to this folder. Once the default
+      configuration is populated, the subsequent times the Devo NG-Relay is run
+      nothing will be populated unless the folder is empty.
+    * *buffer*: Stores the Devo NG-Relay buffer.
+    * *logs*: Stores the Devo NG-Relay logs.
+* All the above folders must have *write* and *execute* permissions for *others*.
 
-The first time when your run the docker Relay, remember to active the relay in Devo Website
+An example of the last two points may be:
 
-Clone or [download](https://github.com/DevoInc/devo-relay/archive/master.zip) the repository.
+```bash
+# Create the folders
+mkdir conf buffer logs
 
-Be sure that [conf/docker-start](./conf/docker-start) has executable permissions after uncompress or clone the repository
+# Grant write and execution for 'others'
+sudo chmod 777 conf/ buffer/ logs/
+```
 
-## Needs to run relay: (Modify in docker-devo-relay.yml)
-1. LOGTRUST_WEB = https://eu.devo.com | https://us.devo.com | https://es.devo.com | ... (Devo web)
-2. LOGTRUST_ENDPOINT = eu.elb.relay.logtrust.net | us.elb.relay.logtrust.net | ... (Devo elb)
-3. LOGTRUST_PORT = 443 | 6514 | 877 | ... (listening port of Devo elb)
-4. RELAY_NAME = The_name_for_your_relay
-5. RELAY_APIKEY = YOUR_API_KEY from your domain when you want to send events
-6. SELFSIGNED=1 Uncomment this environtment variable in [docker-devo-relay.yml](./docker-devo-relay.yml) if you are connecting the relay to endpoint/web with selfsigned certificates
+## Environment
 
-## Run and Stop docker Relay
+To work with this docker-compose file, some environment variables need to be
+set. For this purpose, there is an environment file `.env` in which you can set
+them. There is a short description of each of them below:
 
-1. Run: `DCKHUB=devoinc/ docker-compose -f docker-devo-relay.yml up -d`
-2. Stop: `docker-compose -f docker-devo-relay.yml down`
+| Name                  | Mandatory | Default Value | Description |
+| --------------------- | :-------: | :-----------: | ----------- |
+| NG_RELAY_VERSION      | Yes       | 2.0.1         | Version of the Devo NG-Relay. |
+| LOG_LEVEL             | No        | info          | Sets the log level for the Devo NG-Relay. |
+| JAVA_OPTS             | No        |               | Space-separated quoted list in which you can activate/deactivate any of the JVM flags. |
```

Comment:
Default value should be `-Xmx1G -Xms1G`, right?

Reply:
Right. I will change that.